lib/mayaUsd/fileio/utils/readUtil.cpp

```diff
@@ -150,6 +151,16 @@ _FindOrCreateMayaNumericAttr(
             attr.setUsedAsColor(true);
         }
 
+        const unsigned int numChildren = MFnCompoundAttribute(attrObj).numChildren();
+        if(numChildren < 4u) {
```

Comment:
This can't be right, you must be off by one.  If your tests didn't catch this, then I guess we're missing a bit of coverage :)

Reply:
Ehh...late-night coding. Of course, it can't be right.

Reply:
Done - https://github.com/Autodesk/maya-usd/pull/838/commits/0dd8d8a59e58f46af60ecb49b4091b50ec0a64fb